app/pb_kits/playbook/pb_radio/radio.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Playbook
```

Comment:
The structure of the kit class has been updated, please see [checkbox kit](https://github.com/powerhome/playbook/blob/master/app/pb_kits/playbook/pb_checkbox/checkbox.rb) for example.

Please note the `to_partial_path` and `classname` methods are different.